components/waitwhile/actions/add-customer-note-entry/add-customer-note-entry.ts

```diff
@@ -0,0 +1,52 @@
+import waitwhile from "../../app/waitwhile.app";
+import { defineAction } from "@pipedream/types";
+
+export default defineAction({
+  name: "Add Customer Note Entry",
+  version: "0.0.1",
+  key: "add-customer-note-entry",
```

Comment:
The keys should start with the app name, followed by the name of the component.
```suggestion
  key: "waitwhile-add-customer-note-entry",
```